### PR TITLE
Updated Hindi Translation and corrected typo

### PR DESCRIPTION
**Pull request by SinghRajenM**
* Updated Hindi Translation

* Corrected typo:
Occurenc => Occurrence
and 
Enable with all opened files => Enable for all opened files

### CODE REVIEW

File: PowerEditor/installer/nativeLang/english.xml [hunk 1]
```diff
@@ -151,7 +151,7 @@
                     <Item id="42050" name="Paste Binary Content"/>
                     <Item id="42037" name="Column Mode..."/>
                     <Item id="42034" name="Column Editor..."/>
-                    <Item id="42078" name="Multi-Select Next Occurenc"/>
+                    <Item id="42078" name="Multi-Select Next Occurrence"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/english.xml]: No need to fix this typo, this line will be removed

Comment (SinghRajenM) [reply PowerEditor/installer/nativeLang/english.xml]: Does it mean I need to remove from Hindi.xml too?

Comment (SinghRajenM) [reply PowerEditor/installer/nativeLang/english.xml]: As the line will be removed from English.xml, there is no point to add translation for Hindi.xml too. So removed the typo correction and removed respective line from Hindi.xml too.

**Comment by SinghRajenM** [comment]
@donho Done!!!

**Pull request closed without merging**